### PR TITLE
Run3-alca211A Minor changes in the cfg's used for isotrack calibration of HCAL in Calibration/HcalCalibAlgos

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Minor changes in the cfg's used for isotrack calibration of HCAL in Calibration/HcalCalibAlgos

#### PR validation:

Use the modified cfg's and macros to test run data (pp collision at 900 Gev)

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

File: Calibration/HcalCalibAlgos/test/python/isoTrackNoHLTAnalysis_cfg.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+process = cms.Process("ANALYSIS",Run2_2018)
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.load('Configuration.StandardSequences.Services_cff')
+process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load("RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']
```
Comment (tvami) [inline Calibration/HcalCalibAlgos/test/python/isoTrackNoHLTAnalysis_cfg.py:13]: this seems like a wrong Global tag to use, unless you are simulating 2016

**Comment by tvami** [comment]
@cmsbuild , please test

File: Calibration/HcalCalibAlgos/test/python/isoTrackNoHLTAnalysis_cfg.py (hunk 1 above)
Comment (tvami) [reply Calibration/HcalCalibAlgos/test/python/isoTrackNoHLTAnalysis_cfg.py:13]: @bsunanda FYI

**Comment by tvami** [comment]
+alca
 * tests pass
 * minor comment here https://github.com/cms-sw/cmssw/pull/36181#pullrequestreview-811313465 (should not stop integration)

**Comment by qliphy** [comment]
+1

**Pull request merged**